src/components/contactForm/ContactForm.js

```diff
@@ -1,8 +1,6 @@
 import React from "react";
-
-export const ContactForm = ({
-  contacts,
-  name,
+ const ContactForm = ({
+  cname,

```

Comment:
Please don't change the name value to cname - this will affect other code through the app.

Reply:
changed cname to name in ContactPage.js and ContactFrom.js